tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -2184,13 +2211,32 @@ <h2>Create a distance feature</h2>
         return features;
       },
 
+      strStartsWith_: function(inStr, prefixStr){
+        return inStr.lastIndexOf(prefixStr, 0) == 0;
+      },
+
+      strWithModelIndex_: function(inStr, modelNumber){
+        if (this.getNumberOfModels_() > 1){
+          return inStr + ' ' + (modelNumber + 1);
+        } else {
+          return inStr
+        }
+      },
+
+      getNumberOfModels_: function(){
+        return this.modelName.split(',').length;
+      },
```

Comment:
its strange that each time we want this we have to do this string split. i think numModels should be a data member that is set/updated at any appropriate time during inference or after model name setting, and then is used without needing recalcuating.

Reply:
You are right, moved it to variable numModels which gets updated when user clicks Accept in settings pane.